packages/next-urql/src/with-urql-client.tsx

```diff
@@ -35,7 +37,7 @@ export function withUrqlClient(
           return urqlClient;
         }
 
-        const ssr = ssrExchange({ initialState: urqlState });
+        if (!ssr) ssr = ssrExchange({ initialState: urqlState });
```

Comment:
Wouldn't this then also be reused on the server-side?

Reply:
Seems so yes, I might've worded this wrongly 

```
  const isServer = typeof window === 'undefined';
  if (isServer || !urqlClient) {
    ssrCache = ssrExchange({ initialState });

    urqlClient = createClient({
      ...clientOptions,
      suspense: isServer || clientOptions.suspense,
      exchanges: mergeExchanges(ssrCache),
    });
  }
```

Reply:
Looks okay to me with the server check